torch/signal/windows/windows.py

```diff
@@ -759,3 +732,71 @@ def general_hamming(M,
                           layout=layout,
                           device=device,
                           requires_grad=requires_grad)
+
+
+@_add_docstr(
+    r"""
+Computes the minimum 4-term Blackman-Harris window according to Nuttall.
+
+.. math::
+    w_n = 1 - 0.3635819 \cos{(z)} + 0.4891775 \cos{(2z)} - 0.1365995 \cos{(3z)} + 0.0106411 \cos{(4z)}
```

Comment:
```suggestion
    w_n = \sum_{j=0}^{M - 1} 1 - 0.3635819 \cos{(z_j)} + 0.4891775 \cos{(2z_j)} - 0.1365995 \cos{(3z_j)} + 0.0106411 \cos{(4z_j)}
```

Reply:
nb. If it doesn't fit the formula in the width of the page, perhaps you can trim the last 1 or 2 digits from the decimal expansion of the coefficients?

Reply:
Trimmed 2 and now it fits. Let's see what the build renders.